impl/src/test/groovy/com/okta/sdk/impl/ds/JacksonMapMarshallerTest.groovy

```diff
@@ -121,4 +126,50 @@ class JacksonMapMarshallerTest {
             assertEquals e.getMessage(), "Unable to convert InputStream String to Map."
         }
     }
+
+    @Test
+    void testMapContainingList() {
+
+        def testMap = [
+                "test1": "value1",
+                "simpleListKey": ["one", "two", "three"],
+                "complexResourceList": [new FooResource(null, [foo: "bar1"]),
+                                        new FooResource(null, [foo: "bar2"])]
+        ]
+
+        String result = new JacksonMapMarshaller().marshal(testMap)
+        assertThat result, not(containsString("dirty"))
+        assertThat result, containsString('"complexResourceList":[{"foo":"bar1"},{"foo":"bar2"}]')
```

Comment:
does the result contain `test1` and `simpleListKey` data as well? If so, should we assert them?

Reply:
done